azclishell/app.py

```diff
@@ -387,9 +387,9 @@ def example_repl(self, text, example, start_index):
     def _special_cases(self, text, cmd, outside):
         break_flag = False
         continue_flag = False
-        if 'az' in text:
+        if text and text.split()[0] == 'az':
             telemetry.track_ssg('az', text)
-            cmd = cmd.replace('az', '')
+            cmd = ' '.join(text.split()[1:])
```

Comment:
One thing to consider here is whether you want to strip az at all. If my scope is set to `az network>>` should you strip az?  If so, why would you not strip `az network`? 

We talked about having a way to "break out" of the scope without changing it--perhaps starting your command with az does just that... like specifying an absolute path instead of a relative one.